docs/getting-started.md

```diff
@@ -58,17 +66,18 @@ to [the AKS installation guide](aks-installation.md).
 
 ## Importing Cloud VMs
 
-To manage security policies of Public Cloud VMs, we need to first import
-the target VMs onto `Nephe Controller`. The following section explains
-how to set up access to public cloud account, select target VMs, and
-import VMs into the K8s cluster as VirtualMachine CRs.
+To manage security policies of Public Cloud VMs, we need to first import the
+target VMs onto the `Nephe Controller`. The following section explains how to
+set up access to public cloud account, select target VMs, and import VMs into
+the K8s cluster as `VirtualMachine` CRs.
 
 ### CloudProviderAccount
 
 To import cloud VMs, user needs to configure a `CloudProviderAccount` CR, with
```

Comment:
"Before importing cloud VMs, you need to.."?

Reply:
Done with slight modification. I choose to use `we` as I think that sounds more friendly.